app/assets/javascripts/templates/proceeding_result_template.js

```diff
@@ -0,0 +1,43 @@
+
```

Comment:
[Ditto](https://github.com/ministryofjustice/laa-apply-for-legalaid-api/pull/33/files#r226589071)